keras_app/views/layers_import.py

```diff
@@ -465,6 +465,12 @@ def Padding(layer):
     params = {'pad': pad}
     return jsonLayer('Pad', params, layer)
 
+def TimeDistributed(layer):
+    return jsonLayer('TimeDistributed', {}, layer)
+
+def Bidirectional(layer):
+    params = {}
```

Comment:
Here we need to read the parameters associated with the Bidirectional layer, see `Conv` or other layers for a sample.

Reply:
Done.